lib/helpers.js

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
this file isn't required. 
There is one reference in `ui/lock` which can safely be replaced by `util/url` : 
From : 
```js
define(['helpers', function(helpers){ 
   helpers._url('release','Lock','tao');
});
```
To
```js
define(['util/url', function(urlUtil){ 
   urlUtil.route('release','Lock','tao');
});
```